public/index.html

```diff
@@ -178,6 +178,7 @@ <h2>Test content</h2>
             text: '#d6d6e7',
             colorSourceIcon: '#d6d6e7'
           },
+          detachedMediaQuery: '',
```

Comment:
Can we make that clearer by using true or not empty string?
or 
```json
{ "detached": true | false, "mediaQuery": string }
```

Reply:
detachedMediaQuery: true sounds not very clear neither.
`{ "detached": true | false, "mediaQuery": string }` would need a parent property
Or `detached: true | { mediaQuery: string }`
We can discuss it tomorrow.

Reply:
`detached: true | { mediaQuery: string }` seems good to me